Wabbajack/Util/SystemParametersConstructor.cs

```diff
@@ -56,6 +56,7 @@ public static SystemParameters Create()
                 ScreenHeight = height,
                 VideoMemorySize = video_memory,
                 SystemMemorySize = (long)memory.ullTotalPhys,
+                SystemPageSize = (long)memory.ullTotalPageFile - (long)memory.ullTotalPhys
```

Comment:
Reported pagefile includes physical, I've just subtracted that so we're left with the true amount.